blazor/scheduler/working-hours.md

```diff
@@ -102,7 +102,7 @@ Here, the working days are defined as [1, 3, 4, 5] on Scheduler and therefore th
 
 It is possible to show the week number count of a week in the header bar of the Scheduler by setting true to the `ShowWeekNumber` property. By default, its default value is `false`. In Month view, the week numbers are displayed as a first column.
 
-N> The `ShowWeekNumber` property is not applicable on Timeline views, as it has the equivalent [HeaderRows](./header-rows/#display-week-numbers-in-timeline-views) property to handle such requirement with additional customization.
+N> The `ShowWeekNumber` property is not applicable on Timeline views, as it has the equivalent [HeaderRows](./header-rows#display-week-numbers-in-timeline-views) property to handle such requirement with additional customization.
```

Comment:
why this link alone not updated full link?

Reply:
This link was working but it was redirected. So, I'm updating the link.